reference/promise-types/files.markdown

```diff
@@ -1404,6 +1404,30 @@ comparison or linking operations).
 
 **See also:** [Common Body Attributes][Promise Types and Attributes#Common Body Attributes]
 
+### content
+
+**Description:** Content to write into file
+
+**Type:** `string`
+
+**Allowed input range:** (arbitrary string)
+
+**Example:**
+
+```cf3
```

Comment:
Instead of embedding a snippet example like this, I thinkit's better to dynamically include the example from core.

https://github.com/cfengine/documentation/blob/master/README.md#quoting-policy-files

e.g. https://github.com/cfengine/documentation/blob/master/reference/promise-types/files.markdown#missing_ok

Reply:
Thanks for this tip! I wasn't aware of it. I will rework.

Reply:
Fixed :+1: